### PR TITLE
Revise pull request template

### PR DESCRIPTION
**Pull request by underyx**
### Check list

- [x] I have rebased properly.
      [Please see this for a tutorial.](http://coala.readthedocs.io/en/latest/Developers/Git_Basics.html#rebasing)
- [x] I have gone through the
      [commit guidelines](http://coala.readthedocs.io/en/latest/Developers/Writing_Good_Commits.html)
      and I've followed them:
 - [x] I have followed the
       [guidelines for the commit shortlog](http://docs.coala.io/en/latest/Developers/Writing_Good_Commits.html#shortlog).
 - [x] I have followed the
       [guidelines for the commit body](http://docs.coala.io/en/latest/Developers/Writing_Good_Commits.html#commit-body).
 - [ ] I have included the issue URL with the  `Fixes` keyword if the
       commit fixes a *real bug* and the `Closes` keyword if it adds a
       feature/enhancement.
       [See more here.](http://docs.coala.io/en/latest/Developers/Writing_Good_Commits.html#issue-reference)
- [ ] I have [run all the tests](http://coala.readthedocs.io/en/latest/Developers/Executing_Tests.html)
      and they all pass. If any CI (below) is not green, please fix them. If
      GitMate issues appear you will have to amend your commits, pushing new
      commits on top is not sufficient!

### Reviewers

@sils

Fixed typos, formatting, wording, etc.

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,41 +1,26 @@
 <!--
 Thanks for your contribution!
```
Comment (Adrianzatreanu) [inline PULL_REQUEST_TEMPLATE.md]: i love the changes, but your commit message cooouuld use some work. make it

Revise ``Pull Request`` template
Fixes typos, formatting and wording.

Comment (Adrianzatreanu) [inline PULL_REQUEST_TEMPLATE.md]: note the doubl backticks in which "Pull Request" is enclosed.

Comment (underyx) [inline PULL_REQUEST_TEMPLATE.md]: Why are we treating the phrase 'pull request' as some sort of a proper noun or something? Why capitalize it and wrap it in backticks?

Comment (underyx) [inline PULL_REQUEST_TEMPLATE.md]: I also disagree with the removal of that comma; I'm pretty sure

> Fixes typos, formatting and wording

should be

> Fixes typos, formatting, and wording

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,41 +1,26 @@
...
 
-Reviewing pull requests take really a lot of time and we're all volunteers.
-Please make sure you go through the following check list and complete
-them all before pinging someone for a review.
+Reviewing pull requests takes a lot of time and we're all volunteers. Please make sure you go through the following checklist and all items before pinging someone for a review.
```
Comment (SanketDG) [inline PULL_REQUEST_TEMPLATE.md]: This was wrapped within 80 chars, keep that :+1:

Comment (SanketDG) [inline PULL_REQUEST_TEMPLATE.md]: Oh nvm, I see that's the point of the PR.

Comment (underyx) [inline PULL_REQUEST_TEMPLATE.md]: @sils check the description of this pull request and you'll see :D

**Comment by sils** [comment]
ack https://github.com/coala/coala/pull/2970/commits/6bf107a23a4f8b6e35be7dde66bd3027edb86d68

**Comment by sils** [comment]
rebase and merge

**Comment by underyx** [comment]
reack https://github.com/coala/coala/pull/2970/commits/b73329642725dd090318bbf70e05979b86ad04ee

**Comment by underyx** [comment]
@rultor merge

**Comment by underyx** [comment]
@rultor merge

**Pull request merged**